packages/react-components/react-toast/src/components/Toast/useToastStyles.styles.ts

```diff
@@ -1,19 +1,26 @@
-/**
- * ⚠️ This is temporary and WILL be removed
- */
-
-import { makeStyles, shorthands } from '@griffel/react';
+import { makeStyles, mergeClasses, shorthands } from '@griffel/react';
 import { tokens } from '@fluentui/react-theme';
+import type { ToastSlots, ToastState } from './Toast.types';
+import type { SlotClassNames } from '@fluentui/react-utilities';
 
-export const useToastStyles = makeStyles({
-  toast: {
-    pointerEvents: 'all',
+export const toastClassNames: SlotClassNames<ToastSlots> = {
+  root: 'fui-Toast',
+  timer: 'fui-Timer',
```

Comment:
fui-ToastTimer

Reply:
BTW, is it used at all?

Reply:
no, but this component in general is not public, I'm just following our slot architecture

Reply:
I see, then at least please rename the class as it's a slot of a component